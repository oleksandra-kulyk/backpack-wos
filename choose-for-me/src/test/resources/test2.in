22
2
c3 7 REGULAR 1
c1 3 REGULAR 1
c7 3 SPECIAL 2
c4 8 SPECIAL 1
c2 5 REGULAR 2
c5 6 SPECIAL 1