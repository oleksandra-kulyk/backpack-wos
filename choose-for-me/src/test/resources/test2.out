c1 1
c7 1
c5 1
c2 2